astropy/coordinates/builtin_frames/teme.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+A coordinate or frame in the TEME (True Equator, Mean Equinox) system.
+
+TEME is a True equator, Mean Equinox coordinate frames used in NORAD TLE
+satellite files.
+These are distinct from the ICRS and AltAz frames because they are just
```

Comment:
I'm confused about this comment!

Reply:
I'll just delete it. I think it's a hangover from when that comment was in the transformation file.